opengever/maintenance/scripts/activate_solr.py

```diff
@@ -80,13 +80,24 @@ def remove_catalog_indexes(portal):
             logger.info('Removed catalog index %s.', index)
 
     catalog.clearIndex('Title')
+    title_index = catalog._catalog.getIndex('Title')
+
+    if 'searchable_filing_no' in catalog._catalog.indexes:
+        searchable_filing_no_index = catalog._catalog.getIndex('searchable_filing_no')
+    else:
+        searchable_filing_no_index = None
+    if searchable_filing_no_index:
+        catalog.clearIndex('searchable_filing_no')
```

Comment:
I guess you could move this up into the previous if (`if 'searchable_filing_no' in catalog._catalog.indexes:`)... but it's fine like that

Reply:
oh yes indeed, i moved things around a bit in the end and missed that. thanks!